frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_action.dart

```diff
@@ -22,6 +25,8 @@ enum SmartEditAction {
         return '$input\n\nTl;dr';
       case SmartEditAction.fixSpelling:
         return 'Correct this to standard English:\n\n$input';
+      case SmartEditAction.improveWriting:
+        return 'Rewrite this:\n\n$input';
```

Comment:
Do we not want to make it more implicit that the writing should be improved? Or is that already default behavior.

Eg. `Improve the writing of:`

Reply:
I think giving this prompt gets the desired result